packages/atlas/src/views/studio/MyUploadsView/UploadStatus/UploadStatus.tsx

```diff
@@ -135,11 +135,16 @@ export const UploadStatus: FC<UploadStatusProps> = ({ isLast = false, asset, siz
     noClick: true,
     noKeyboard: true,
     accept: {
-      [isVideo ? 'video/*' : 'image/*']: [],
+      [asset.type === 'video' ? 'video/*' : asset.type === 'subtitle' ? 'text/vtt' : 'image/*']: [],
     },
   })
 
-  const fileTypeText = isVideo ? 'Video file' : `${asset.type.charAt(0).toUpperCase() + asset.type.slice(1)} image`
+  const fileTypeText =
+    asset.type === 'video'
+      ? 'Video file'
+      : asset.type === 'subtitle'
+      ? 'Subtitle'
```

Comment:
```suggestion
      ? 'Subtitles'
```
I would also change `asset.type` as I mentioned in one of my comment above

Reply:
Done